src/DashTextareaAutocomplete.jl

```diff
@@ -16,14 +16,14 @@ function __init__()
             [
                 DashBase.Resource(
     relative_package_path = "dash_textarea_autocomplete.min.js",
-    external_url = "https://unpkg.com/dash_textarea_autocomplete@1.1.0/dash_textarea_autocomplete/dash_textarea_autocomplete.min.js",
+    external_url = "https://unpkg.com/dash-textarea-autocomplete@1.1.0/dash_textarea_autocomplete/dash_textarea_autocomplete.min.js",
```

Comment:
Unfortunately, `npm run build` (required to bundle the JS files and adapt the py,jl,R files accordingly) will override these modifs

<details>
<summary>Demo, click me</summary>

```
(venv) ✔ ~/Documents/repos/dash-components/dash-textarea-autocomplete [tweak-external-urls-and-gitignore|✔] 
09:43 $ npm run build

> dash-textarea-autocomplete@1.1.0 build /home/etienne/Documents/repos/dash-components/dash-textarea-autocomplete
> npm run build:js && npm run build:backends


> dash-textarea-autocomplete@1.1.0 build:js /home/etienne/Documents/repos/dash-components/dash-textarea-autocomplete
> webpack --mode production

Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
Hash: 5242f0501bb5746c47ac
Version: webpack 4.36.1
Time: 2252ms
Built at: 2021-12-09 9:43:45 a.m.
                                Asset      Size  Chunks             Chunk Names
    dash_textarea_autocomplete.min.js  34.2 KiB       0  [emitted]  main
dash_textarea_autocomplete.min.js.map   124 KiB       0  [emitted]  main
Entrypoint main = dash_textarea_autocomplete.min.js dash_textarea_autocomplete.min.js.map
 [0] external "React" 42 bytes {0} [built]
 [1] external "PropTypes" 42 bytes {0} [built]
 [2] external "ReactDOM" 42 bytes {0} [built]
 [5] (webpack)/buildin/global.js 472 bytes {0} [built]
[11] ./src/lib/index.js + 2 modules 55.8 KiB {0} [built]
     | ./src/lib/index.js 170 bytes [built]
     |     + 2 hidden modules
    + 7 hidden modules

> dash-textarea-autocomplete@1.1.0 build:backends /home/etienne/Documents/repos/dash-components/dash-textarea-autocomplete
> dash-generate-components ./src/lib/components dash_textarea_autocomplete -p package-info.json --r-prefix '' --jl-prefix ''

Generated DashTextareaAutocomplete.py
Generated dashTextareaAutocomplete.R
Generated dashtextareaautocomplete.jl
Generated src/DashTextareaAutocomplete.jl
Generated Project.toml
(venv) ✔ ~/Documents/repos/dash-components/dash-textarea-autocomplete [tweak-external-urls-and-gitignore|✚ 2] 
09:43 $ gg
## tweak-external-urls-and-gitignore...origin/tweak-external-urls-and-gitignore
 M Project.toml
 M src/DashTextareaAutocomplete.jl
(venv) ✔ ~/Documents/repos/dash-components/dash-textarea-autocomplete [tweak-external-urls-and-gitignore|✚ 2] 
09:44 $ git d
diff --git a/Project.toml b/Project.toml
index 42753c0..064720b 100644
--- a/Project.toml
+++ b/Project.toml
@@ -9,4 +9,4 @@ Dash = "1b08a953-4be3-4667-9a23-3db579824955"
 
 [compat]
 julia = "1.2"
-Dash = "0.1.3, 1.0"
+Dash = "0.1.3"
diff --git a/src/DashTextareaAutocomplete.jl b/src/DashTextareaAutocomplete.jl
index 4438d78..29cdf02 100644
--- a/src/DashTextareaAutocomplete.jl
+++ b/src/DashTextareaAutocomplete.jl
@@ -16,14 +16,14 @@ function __init__()
             [
                 DashBase.Resource(
     relative_package_path = "dash_textarea_autocomplete.min.js",
-    external_url = "https://unpkg.com/dash-textarea-autocomplete@1.1.0/dash_textarea_autocomplete/dash_textarea_autocomplete.min.js",
+    external_url = "https://unpkg.com/dash_textarea_autocomplete@1.1.0/dash_textarea_autocomplete/dash_textarea_autocomplete.min.js",
     dynamic = nothing,
     async = nothing,
     type = :js
 ),
 DashBase.Resource(
     relative_package_path = "dash_textarea_autocomplete.min.js.map",
-    external_url = "https://unpkg.com/dash-textarea-autocomplete@1.1.0/dash_textarea_autocomplete/dash_textarea_autocomplete.min.js.map",
+    external_url = "https://unpkg.com/dash_textarea_autocomplete@1.1.0/dash_textarea_autocomplete/dash_textarea_autocomplete.min.js.map",
     dynamic = true,
     async = nothing,
     type = :js

```
</details>

More info in https://github.com/etpinard/dash-textarea-autocomplete/blob/main/CONTRIBUTING.md

---

What does this mean? Sure if we're in a rush to make `DashTextareaAutocomplete` to work in the compiled Confab, we can merge and release this PR.

**BUT** if we need to make a modification to the JS files at some point **OR** the `dash-generate-components` (defined in the `dash.py` package) build step changes in the future to accommodate a future version of Dash, we would need to _manually_ patch these lines again and remember to do so -- which is usually harder :smirk: 

So, ideally, we should make summit a patch in `dash.py` (most likely for this [line](https://github.com/plotly/dash/blob/51a7db52e448361be7e0d1444235decfe90bbc4a/dash/development/_jl_components_generation.py#L60)). But getting this merged and released might take a while.

Or, add a post-build step in this repo similar to what https://github.com/etpinard/dash-textarea-autocomplete/pull/8/ did. Adding something like

```
sed -i 's/https:\/\/unpkg.com\/dash_textarea_autocomplete/https:\/\/unpkg.com\/dash-textarea-autocomplete/' src/DashTextareaAutocomplete.jl
```

should work fine.

Reply:
Ok, thanks for all the information!

right quickly, before looking on how to follow you recommendations, it looks like we also lose the compat bound for dash `1.0` with `npm run build`

```
-Dash = "0.1.3, 1.0"
+Dash = "0.1.3"
```